rbac/common/sawtooth/state_client.py

```diff
@@ -24,18 +24,17 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class StateClient:
-    def get_address(self, state, address):
-        try:
-            return state.get_state(addresses=[address], timeout=TIMEOUT_SECONDS)
-        except FutureTimeoutError:
-            raise InternalError(ERROR_MESSAGE_TIMEOUT, TIMEOUT_SECONDS)
-
-    def set_address(self, state, address, container):
-        try:
-            return state.set_state(
-                entries={address: container.SerializeToString()},
-                timeout=TIMEOUT_SECONDS,
-            )
-        except FutureTimeoutError:
-            raise InternalError(ERROR_MESSAGE_TIMEOUT, TIMEOUT_SECONDS)
+def get_address(state, address):
+    try:
+        return state.get_state(addresses=[address], timeout=TIMEOUT_SECONDS)
+    except FutureTimeoutError:
+        raise InternalError(ERROR_MESSAGE_TIMEOUT, TIMEOUT_SECONDS)
+
+
+def set_address(state, address, container):
+    try:
```

Comment:
DS

Reply:
It's not worth the time to rebase #536 to add docstrings to #532.
Docstrings will be added in a followup PR.